compiler/src/main/resources/play_scala_security_extractors.mustache

```diff
@@ -8,6 +8,11 @@ import de.zalando.play.controllers.SwaggerSecurityExtractors._
 import {{import.name}}
 {{/for}}
 
+object SecurityExtractorsExecutionContext {
+    // TODO override with proper ExecutionContext instance
+    implicit val ec = scala.concurrent.ExecutionContext.global
```

Comment:
Might make sense, but then you should still create one the Play way I think... 
https://www.playframework.com/documentation/2.4.x/ThreadPools#Using-other-thread-pools
(not sure what the real disadvantage is to use the scala execution context, but the stack overflow answer advises against it)